resources/lib/addon.py

```diff
@@ -15,131 +15,130 @@
     ''' The API interface to unwatch an episode used by the context menu '''
     from resumepoints import ResumePoints
-    ResumePoints(kodi).unwatchlater(uuid=uuid, title=to_unicode(unquote_plus(from_unicode(title))), url=url)
+    ResumePoints(KodiWrapper(globals())).unwatchlater(uuid=uuid, title=to_unicode(unquote_plus(from_unicode(title))), url=url)
 
 
 @plugin.route('/favorites')
 def favorites_menu():
     ''' The My favorites menu '''
     from vrtplayer import VRTPlayer
-    VRTPlayer(kodi).show_favorites_menu()
+    VRTPlayer(KodiWrapper(globals())).show_favorites_menu()
 
 
 @plugin.route('/favorites/programs')
 def favorites_programs():
     ''' The favorites 'My programs' listing '''
     from vrtplayer import VRTPlayer
-    VRTPlayer(kodi).show_tvshow_menu(use_favorites=True)
+    VRTPlayer(KodiWrapper(globals())).show_tvshow_menu(use_favorites=True)
 
 
 @plugin.route('/favorites/docu')
 def favorites_docu():
     ''' The favorites docu listing '''
     from vrtplayer import VRTPlayer
-    VRTPlayer(kodi).show_favorites_docu_menu()
+    VRTPlayer(KodiWrapper(globals())).show_favorites_docu_menu()
 
 
 @plugin.route('/favorites/recent')
 @plugin.route('/favorites/recent/<page>')
 def favorites_recent(page=1):
     ''' The favorites recent listing '''
     from vrtplayer import VRTPlayer
-    VRTPlayer(kodi).show_recent_menu(page=page, use_favorites=True)
+    VRTPlayer(KodiWrapper(globals())).show_recent_menu(page=page, use_favorites=True)
 
 
 @plugin.route('/favorites/offline')
 @plugin.route('/favorites/offline/<page>')
 def favorites_offline(page=1):
     ''' The favorites offline listing '''
     from vrtplayer import VRTPlayer
-    VRTPlayer(kodi).show_offline_menu(page=page, use_favorites=True)
+    VRTPlayer(KodiWrapper(globals())).show_offline_menu(page=page, use_favorites=True)
 
 
 @plugin.route('/favorites/refresh')
 def favorites_refresh():
     ''' The API interface to refresh the favorites cache '''
     from favorites import Favorites
-    Favorites(kodi).refresh(ttl=0)
-    kodi.show_notification(message=kodi.localize(30982))
+    Favorites(KodiWrapper(globals())).refresh(ttl=0)
+    KodiWrapper(globals()).show_notification(message=KodiWrapper(globals()).localize(30982))
 
 
 @plugin.route('/favorites/manage')
 def favorites_manage():
     ''' The API interface to manage your favorites '''
     from favorites import Favorites
-    Favorites(kodi).manage()
+    Favorites(KodiWrapper(globals())).manage()
 
 
 @plugin.route('/resumepoints/continue')
 def resumepoints_continue():
     ''' The resumepoints continue listing '''
     from vrtplayer import VRTPlayer
-    VRTPlayer(kodi).show_continue_menu(page=1)
+    VRTPlayer(KodiWrapper(globals())).show_continue_menu(page=1)
 
 
 @plugin.route('/resumepoints/refresh')
 def resumepoints_refresh():
     ''' The API interface to refresh the resumepoints cache '''
     from resumepoints import ResumePoints
-    ResumePoints(kodi).refresh(ttl=0)
```

Comment:
You will only need to do this once per function.

Reply:
Indeed, I did a quick find and replace. Fixed it.